### DISCUSSION TITLE
My domain registrar says Github Ips are blacklisted

### DISCUSSION THREAD

**Discussion by Abiel0** [community/community#143297] [New to GitHub]
Today I added my domains to domain registrar. But it says The Ip is in a blacklist. I checked a blacklist checker and it said this blacklist is made by spamhaus and to check the reason https://check.spamhaus.org/sbl/query/SBL658687 . I send them a mail explaining These github ips are safe , How can we resolve this problem so I can add my domains to my registrar.

**Comment by ghostinhershell** [comment]
Hi @Abiel0, thanks so much for reaching out in our community and reporting this issue. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.
